### ISSUE TITLE
test_lint.sh: No space left on device

### ISSUE THREAD

**Issue by glasnt** [GoogleCloudPlatform/cloud-foundation-toolkit#2427]
### TL;DR

Recent lint checks in terraform-docs-samples have been failing due to disk space errors

### Expected behavior

Successful linting. 

### Observed behavior

```
parallel: Error: Output is incomplete.
parallel: Error: Cannot append to buffer file in /tmp.
parallel: Error: Is the disk full?
Error: No space left on device
```

### Terraform Configuration

Issues occurring in CI on https://github.com/terraform-google-modules/terraform-docs-samples

### Terraform Version

Whatever is provided in cft/developer-tools:1.21

### Additional information

Failing actions: 

* https://github.com/terraform-google-modules/terraform-docs-samples/actions/runs/9588316133/attempts/1?pr=679
* https://github.com/terraform-google-modules/terraform-docs-samples/actions/runs/9588316133/attempts/2?pr=679
* https://github.com/terraform-google-modules/terraform-docs-samples/actions/runs/9588316133/attempts/3?pr=679

Successful action (after "enable debugging"): 

https://github.com/terraform-google-modules/terraform-docs-samples/actions/runs/9588316133?pr=679

**Comment by glasnt** [comment]
Nevermind, I believe this is related to https://github.com/terraform-google-modules/terraform-docs-samples/pull/672
